my_list.rb

```diff
@@ -0,0 +1,20 @@
+require './my_enumerable'
```

Comment:
- [ ] Kindly use `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention.